DOCUMENTATION.md

```
@@ -1012,27 +959,23 @@ These helper functions facilitate various operations related to job management,
 | Install jellyfin container | install remove purge status help | @armbian 
 | Install jellyseerr container | install remove purge status help | @armbian 
 | Needed by generate_menu | execute_command 'id' | @Tearran 
 | Display a Yes/No dialog box and process continue/exit | get_user_continue 'Do you wish to continue?' process_input | @Tearran 
 | Module for Armbian firmware manipulating. | select install show hold unhold repository headers help | @igorpecovnik 
 | Deploy Armbian KVM instances | install remove save drop restore list help | @igorpecovnik 
-| Install OpenMediaVault (OMV) | install remove status help | @igorpecovnik 
 | Unmask service | srv_unmask ssh.service | @dimitry-ishenko 
 | Migrated procedures from Armbian config. | connect_bt_interface | @armbian 
 | Display a message box | show_message <<< 'hello world'  | @Tearran 
-| Manage self hosted runners | install remove remove_online purge status help | @igorpecovnik 
+| Manage self hosted runners | install remove remove_online purge help | @igorpecovnik 
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)